bazel/grpc_build_system.bzl

```diff
@@ -74,8 +74,8 @@ def grpc_cc_library(
         visibility = None,
         alwayslink = 0,
         data = [],
-        copts = [],
-        linkopts = []):
```

Comment:
Formatting issue? Loses a paren and colon?

Reply:
Rather, syntax issue, not formatting. Build broken.

Reply:
Going too fast.